rsb/introspection/__init__.py

```diff
@@ -726,26 +727,34 @@ def handle_participant_destruction(participant):
         _sender = None
 
 
-def initialize(display_name=None):
+_introspection_initialized = False
+_introspection_mutex = threading.RLock()
+
+
+def rsb_initialize():
     """
     Initialize the introspection module.
 
     Clients need to ensure that this method is called only once.
```

Comment:
Is this still true?

Reply:
I meant the comment as pertaining to the function. It gained a flag and a lock so clients calling it multiple times should be fine now.

Reply:
ok, the highlight is on the lock here, which is confusing. Anyway, I am not exactly sure whether this would really need a lock. Other plugins don't have this. But I will change the comment.

Reply:
I thought it was on the `Clients need to ensure that this method is called only once.` line, sorry.